Apps/Patient/src/Services/PatientService.cs

```diff
@@ -72,14 +72,13 @@ public PatientService(ILogger<PatientService> logger, IPatientRepository patient
             {
                 // BCHCIM.GD.2.0018 Not found
                 this.logger.LogWarning("Client Registry did not find any records. Returned message code: {ResponseCode}", "Not found");
-                throw new ProblemDetailsException(ExceptionUtility.CreateProblemDetails(ErrorMessages.ClientRegistryRecordsNotFound, HttpStatusCode.NotFound, nameof(PatientService)));
+                throw new NotFoundException(ErrorMessages.ClientRegistryRecordsNotFound);
             }
 
             if (patientDetails.IsDeceased == true)
             {
                 this.logger.LogWarning("Client Registry returned a person with the deceased indicator set to true. No PHN was populated. {ActionType}", ActionType.Deceased.Value);
-                throw new ProblemDetailsException(
-                    ExceptionUtility.CreateProblemDetails(ErrorMessages.ClientRegistryReturnedDeceasedPerson, HttpStatusCode.NotFound, nameof(PatientService)));
+                throw new DataMismatchException(ErrorMessages.ClientRegistryReturnedDeceasedPerson);
             }
```

Comment:
Applies to:

- Patient
  - PatientController.GetPatientV2()

These usages of Data Mismatch may better suit a new exception, as discussed above.